src/applications/lifecycles/bootstrap.js

```diff
@@ -15,7 +15,12 @@ export async function toBootstrapPromise(appOrParcel) {
     appOrParcel.status = NOT_MOUNTED;
   } catch(err) {
     appOrParcel.status = SKIP_BECAUSE_BROKEN;
-    handleAppError(err, appOrParcel);
+    if (hardFail) {
+      const transformedErr = transformErr(err, appOrParcel)
+      throw transformedErr
+    } else {
+      handleAppError(err, appOrParcel);
+    }
```

Comment:
The error on the window object and the error that parcels will need to catch are the same now that I'm reusing `transformErr`